ChangesetDiff.py

```diff
@@ -23,30 +27,31 @@ def serialize(self, add, delete):
             if (graphUri in delete.keys() and len(delete[graphUri]) > 0) or (graphUri in add.keys() and len(add[graphUri]) > 0):
                 diff = Namespace("urn:changeset:" + str(uuid.uuid1()))
                 graphTerm = diff.term("")
-                g.add( (graphTerm, RDF.type, changeset.ChangeSet) )
+                g.add((graphTerm, RDF.type, changeset.ChangeSet))
                 if str(graphUri) != 'default':
-                    g.add( (graphTerm, changeset.subjectOfChange, graphUri) )
+                    g.add((graphTerm, changeset.subjectOfChange, graphUri))
                 if graphUri in delete.keys() and len(delete[graphUri]) > 0:
                     i = 0
                     for triple in delete[graphUri]:
                         deleteStatementName = BNode()
-                        g.add( (graphTerm, changeset.removal, deleteStatementName) )
-                        g.add( (deleteStatementName, RDF.type, RDF.Statement) )
-                        g.add( (deleteStatementName, RDF.subject, triple[0]) )
-                        g.add( (deleteStatementName, RDF.predicate, triple[1]) )
-                        g.add( (deleteStatementName, RDF.object, triple[2]) )
+                        g.add((graphTerm, changeset.removal, deleteStatementName))
+                        g.add((deleteStatementName, RDF.type, RDF.Statement))
+                        g.add((deleteStatementName, RDF.subject, triple[0]))
+                        g.add((deleteStatementName, RDF.predicate, triple[1]))
+                        g.add((deleteStatementName, RDF.object, triple[2]))
                         i += 1
                 if graphUri in add.keys() and len(add[graphUri]) > 0:
                     i = 0
                     for triple in add[graphUri]:
                         insertGraphName = BNode()
-                        g.add( (graphTerm, changeset.addition, insertGraphName) )
-                        g.add( (insertGraphName, RDF.type, RDF.Statement) )
-                        g.add( (insertGraphName, RDF.subject, triple[0]) )
-                        g.add( (insertGraphName, RDF.predicate, triple[1]) )
-                        g.add( (insertGraphName, RDF.object, triple[2]) )
+                        g.add((graphTerm, changeset.addition, insertGraphName))
+                        g.add((insertGraphName, RDF.type, RDF.Statement))
+                        g.add((insertGraphName, RDF.subject, triple[0]))
+                        g.add((insertGraphName, RDF.predicate, triple[1]))
+                        g.add((insertGraphName, RDF.object, triple[2]))
                         i += 1
 
         return g.serialize(format="turtle").decode("utf-8")
 
+
```

Comment:
In some cases, yes:
`E305 expected 2 blank lines after class or function definition, found 1 [pep8]`